indico/modules/events/abstracts/templates/_abstract.html

```diff
@@ -0,0 +1,201 @@
+{% macro _render_tracks(tracks) %}
+    {% for track in tracks %}
+        <span class="abstract-track" title="{{ track._description }}">{{ track.title }}</span>
+        {%- if not loop.last -%},{% endif %}
+    {% endfor %}
+{% endmacro %}
+
+
+{% macro render_abstract_public(abstract) %}
+    <div id="abstract-title">
+        <h3 class="abstract-title">
+            {{ abstract.title }}
+            <span class="abstract-id">
+                #{{ abstract.friendly_id }}
+            </span>
+        </h3>
+        <div class="toolbar thin">
+            <div class="group">
+                <a class="i-button icon-edit"></a>
+            </div>
+            <div class="group">
+                <a class="i-button icon-file-download"></a>
+            </div>
+        </div>
+    </div>
+
+    <div id="abstract-public">
+        <div class="abstract-summary">
+            <div class="abstract-summary-state {{ abstract.public_state.name }}">
+                {{ abstract.public_state.title }}
+            </div>
+            <div class="abstract-summary-content">
+                <div>
+                    {% if abstract.submitted_contrib_type %}
+                        {% trans name=abstract.submitter.name, type=abstract.submitted_contrib_type.name|lower %}
+                            <strong>{{ name }}</strong> submitted this <span class="abstract-contribution-type">{{ type }}</span>
+                        {% endtrans %}
+                    {% else %}
+                        {% trans name=abstract.submitter.name %}
+                            <strong>{{ name }}</strong> submitted this abstract
+                        {% endtrans %}
+                    {% endif %}
+                </div>
+                {% if abstract.submitted_for_tracks %}
+                    <div>
+                        {% trans count=abstract.submitted_for_tracks|length %}
+                            For track:
```

Comment:
The idea is to make it look like a sentence, but I have to give another iteration.